app/src/main/res/layout/article_row.xml

```diff
@@ -32,27 +34,52 @@
             android:layout_width="0dp"
             android:layout_height="wrap_content"
             android:layout_margin="4dp"
+            android:layout_marginTop="8dp"
             android:padding="4dp"
+            android:paddingStart="6dp"
+            android:paddingEnd="6dp"
             android:textColor="#000"
-            android:textSize="16sp"
+            android:textSize="22sp"
             android:textStyle="bold"
-            app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintStart_toEndOf="@id/photo"
-            app:layout_constraintTop_toTopOf="parent" />
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/photo" />
 
         <TextView
             android:id="@+id/snippet"
             android:layout_width="0dp"
             android:layout_height="wrap_content"
             android:layout_margin="4dp"
+            android:layout_marginBottom="8dp"
             android:ellipsize="end"
             android:maxLines="3"
             android:padding="4dp"
             android:textColor="#000"
             android:textSize="16sp"
-            app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintStart_toEndOf="@id/photo"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toBottomOf="@id/title" />
+
+        <android.support.v7.widget.CardView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            app:layout_constraintBottom_toBottomOf="@id/photo"
+            app:layout_constraintLeft_toLeftOf="@id/photo"
+            android:layout_margin="5dp"
+            app:cardBackgroundColor="#ffffff"
+            app:cardCornerRadius="7dp">
+
+            <TextView
+                android:id="@+id/sentiment"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:background="@drawable/rounded_corner"
```

Comment:
`@drawable/rounded_corner` on TextView isn't needed anymore, can be removed now that you have it in a `CardView`

Reply:
Also can do the Code -> Reformat Code thing in this article_row.xml